appveyor-run.cmd

```diff
@@ -7,14 +7,15 @@
 ::
 SETLOCAL EnableDelayedExpansion
 
-SET counter=0
+SET operation=%1
+SET runtime=%2
+SET toolkit=%3
+
+CALL edm run -- python etstool.py !operation! --runtime=!runtime! --toolkit=!toolkit! || GOTO error
+PUSHD ets-demo
+CALL edm run -- python etstool.py !operation! --runtime=!runtime! --toolkit=!toolkit! || GOTO error
+POPD
 
-FOR %%x IN (%*) DO (
-SET /A counter=!counter! + 1
-IF !counter! EQU 1 SET operation=%%x
-IF !counter! EQU 2 SET runtime=%%x
-IF !counter! GTR 2 CALL edm run -- python etstool.py !operation! --runtime=!runtime! --toolkit=%%x || GOTO error
-)
```

Comment:
Drive-by structural changes to the batch script. The original script took me a while to understand. I don't know what called for the initial indirection, but whatever it was trying to workaround does not seem to apply now.

Reply:
Post-mortem: turns out the loop here is such that we can run this batch script like this:
```
appveyor-run.cmd install "3.6" null pyqt pyqt4 wx
```
The change here means the rest of the toolkit after the first one is ignore.
#971 sets the toolkit argument to a single value in the build configuration.